### DISCUSSION TITLE
The selected folder does not contain a valid Adobe AIR SDK

### DISCUSSION THREAD

**Discussion by pcrivatz** [airsdk/Adobe-Runtime-Support#645] [Q&A]
Can't add the adobe air sdk to adobe animate. I am trying to add the latest version from harman, Adobe AIR - version 33.0.2.338. I am using Adobe Animate 2020.
I tried reinstalling Adobe Animate, it didn't work. I can't add any version of AIR SDK.
![image](https://user-images.githubusercontent.com/40854507/73848968-c7266100-4831-11ea-96ff-95cf85eb797d.png)

**Comment by itlancer** [comment]
Maybe you have downloaded AIR SDK for macOS?

**Comment by pcrivatz** [comment]
No, it was for windows, I am sure, and none of the AIR SDK's for windows work, if I try to add them in Animate CC 2019 it works, but in Animate 2020 it doesn't.

**Comment by ghost** [comment]
Animate 2020 + AIR 33.0.2.338 works for me, I'm wondering though whether this might be Java related. If you open up a new cmd prompt, and do "java -version", can you confirm what you get? And then if you go into the folder where the SDK is and into its "bin" folder, and run "adt -version", does that work?

**Comment by ajwfrost** [comment]
Hi - did you get this working? I can double-check with the Animate team to see exactly what they test for in order to validate the SDK folder. I'm hoping it's just that they run "adt -version" on it, but they may also look for the air-sdk-description.xml file so perhaps check that this is there and is in the right format?

thanks
